### DISCUSSION TITLE
Datahub Integration - Spark

### DISCUSSION THREAD

**Discussion by babjiloganda** [great-expectations/great_expectations#8141] [Community Use Cases]
Hi Team,

Do we have a solution for integrating spark datasource with GE and Datahub. Currently I see there is support for SQLAlchemy (https://docs.greatexpectations.io/docs/integrations/integration_datahub/#further-discussion). If you are bringing in the integration of Spark datasource in future kindly let us know.

Thanks,
Babji

**Comment by Stefs-2142** [comment]
There opened FR for this topic - https://feature-requests.datahubproject.io/p/great-expectations-support-different-execution-engines
